tests/factories.py

```diff
@@ -549,6 +554,36 @@ class ArchiveTargetFactory(ModularOdmFactory):
 class ArchiveJobFactory(ModularOdmFactory):
     FACTORY_FOR = ArchiveJob
 
+
+class DraftRegistrationFactory(ModularOdmFactory):
+    FACTORY_FOR = DraftRegistration
+
+    @classmethod
+    def _create(cls, *args, **kwargs):
+        branched_from = kwargs.get('branched_from')
+        initiator = kwargs.get('initiator')
+        registration_schema = kwargs.get('registration_schema')
+        registration_metadata = kwargs.get('registration_metadata')
+        if not branched_from:
+            project_params = {}
+            if initiator:
+                project_params['creator'] = initiator
+            branched_from = ProjectFactory(**project_params)
+        initiator = branched_from.creator
+        try:
+            registration_schema = registration_schema or MetaSchema.find()[0]
+        except IndexError:
+            ensure_schemas()
+        registration_metadata = registration_metadata or {}
+        draft = branched_from.create_draft_registration(
```

Comment:
Minor: I think it makes sense to have `MetaSchema.create_from_node(node)` rather than `Node.create_draft_registration(...)`. Ideally, `Node` wouldn't have to know anything about `MetaSchema`.

I think this is fine for now, though.